hybris/properties/hybris_properties.c

```diff
@@ -61,20 +68,29 @@ int property_list(void (*propfn)(const char *key, const char *value, void *cooki
 {
     ensure_bionic_properties_initialized();
 
-    return bionic_property_list(propfn, cookie);
+    if (!own_impl)
+        return bionic_property_list(propfn, cookie);
+    else
+        return my_property_list(propfn, cookie);
 }
 
 int property_get(const char *key, char *value, const char *default_value)
 {
     ensure_bionic_properties_initialized();
 
-    return bionic_property_get(key, value, default_value);
+    if (!own_impl)
+        return bionic_property_get(key, value, default_value);
+    else
+        return my_property_get(key, value, default_value);
 }
 
 int property_set(const char *key, const char *value)
 {
     ensure_bionic_properties_initialized();
 
-    return bionic_property_set(key, value);
+    if (!own_impl)
+         return bionic_property_set(key, value);
```

Comment:
Indentation seems to be off here (tabs vs spaces?)

Reply:
sorry for the nitpick, can you fix the indentation here? then i'll merge it

Reply:
Oh sorry, this got buried in my email. Will fix it right away.